docs/01_nodeos/02_usage/03_development-environment/index.md

```diff
@@ -10,12 +10,18 @@ This is the go-to option for smart contract developers, aspiring Block Producers
 
 * [Configure Nodeos as a Local Single-node Testnet](00_local-single-node-testnet.md) 
 
+## Local Single-Node Testnet With Consensus Protocol
+
+This is the go-to option for smart contract developers, aspiring Block Producers or Non-Producing Node operators. It has the most simple configuration with the least number of requirements to setup a local single-node testnet with consensus protocol enabled.
+
+* [Configure Nodeos as a Local Single-node Testnet with Consensus Protocol](10_local-single-node-testnet-consensus.md)
+
 ## Local Multi-Node Testnet
 
 While this option can technically be used for smart contract development, it may be overkill. This is most beneficial for those who are working on aspects of core development, such as benchmarking, optimization and experimentation. It's also a good option for hands-on learning and concept proofing.
```

Comment:
Suggestion
The word overkill sounds too informal? How about:

This is an advanced configuration option compared to the Local Single-Node Testnet....

This option is one level advanced than the Local Single-Node Testnet....

Just a suggestion...

Reply:
I don't find it too informal. We are developers after all.